test/README.md

```diff
@@ -22,6 +22,7 @@ The E2E tests run against local deployments of the parachain, relayer, the ether
 * Development environment for the relay services. See setup [instructions](../relayer/README.md#development).
 * `jq` - https://stedolan.github.io/jq/download/
 * geth - https://geth.ethereum.org/docs/install-and-build/installing-geth
+* g++ (required for lodestar) is available in the `build-essential` package: `apt install build-essential`
```

Comment:
Interesting! I'm trying to remember if I had to install this. 🤔

Reply:
It wasn't installed on my fresh instance, but you may well have installed it as part of the setup steps for something else.

Reply:
Installing `build-essential` is part of the parachain setup documentation in `parachain/README.md`.
See https://docs.substrate.io/install/linux/